src/utils.rs

```diff
@@ -20,12 +20,13 @@ use merlin::Transcript;
 use rand::{CryptoRng, Rng, RngCore};
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use std::fmt::Debug;
+use zeroize::Zeroize;
 
 pub(crate) const CRYPTOGRAPHIC_RETRY_MAX: usize = 500usize;
 
 /// Wrapper around k256::ProjectivePoint so that we can define our own
 /// serialization/deserialization for it
-#[derive(Eq, PartialEq, Debug, Clone, Copy)]
+#[derive(Eq, PartialEq, Debug, Clone, Copy, Zeroize)]
```

Comment:
Should this be `Debug`? I guess it's very context dependent whether the point is private or not.

Reply:
Yeah, I looked through the uses here and the privacy varies. I decided to make this `Debug` and require any wrapper types to specifically redact it if necessary (e.g. `PresignRecord` does, `round_three::Private` doesn't).